### DISCUSSION TITLE
希望直播界面可选择隐藏下方礼物条，类似网页版网页全屏

### DISCUSSION THREAD

**Discussion by Yumeyo23** [ywmoyue/biliuwp-lite#486] [feature]
### 问题描述

目前默认的直播界面下面有礼物条，如果铺满窗口又看不到右侧弹幕，这样默认小窗模式很不爽，希望能可选隐藏掉礼物条，跟网页直播的网页全屏的时候一样，
<img width="1280" alt="PixPin_2024-01-30_04-31-21" src="https://github.com/ywmoyue/biliuwp-lite/assets/88522326/4b508c52-2cd8-4670-950f-d5a7f450a765">
<img width="1280" alt="PixPin_2024-01-30_04-40-10" src="https://github.com/ywmoyue/biliuwp-lite/assets/88522326/bba44e6a-409d-4212-8488-858b46cbb958">

（我咋记得以前好像默认是隐藏的？）

### 描述解决方案

介绍如上，已更新到目前公开最新版，软件挺好用的，感谢软件，加油

### 附加信息

已搜索历史issue未找到类似建议
抱歉不会编程qwq只能能提建议了望采纳

### 代码贡献

否

**Comment by GD-Slime** [comment]
一直有礼物栏确实不太好看。 
@ywmoyue 添加一个类似于b站网页端的图标在播放器控制栏： 
![image](https://github.com/ywmoyue/biliuwp-lite/assets/82302542/736a13c3-7903-4e1a-8784-53e39e2ff30f)
来关闭 / 开启下面的整个礼物部分
这样是否可行？

**Reply by ywmoyue** [reply]
可行

**Reply by ToumaLover** [reply]
请问可否增加一个设置项来开关礼物部分，或者记忆礼物栏关闭按钮的状态

**Reply by ywmoyue** [reply]
@ToumaLover 这个discussion已关闭了，直接去提新的需求issue吧 https://github.com/ywmoyue/biliuwp-lite/issues/new/choose
